wsl-pro-service/internal/system/system_test.go

```diff
@@ -399,6 +399,8 @@ func TestLandscapeEnable(t *testing.T) {
 	}{
 		"Success":                                    {},
 		"Success overriding computer_title":          {},
+		"Success when empty tags are present":        {},
+		"Success when custom tags are present":       {},
```

Comment:
nitpick: I would change the name here to be more explicit than just "Success", like "Do not append wsl when config tag is provided but empty" and "Do not append wsl tag when config tag is provided"